src/components/LocationAutocomplete/LocationAutocomplete.tsx

```diff
@@ -199,23 +199,23 @@ const LocationAutocomplete: React.FC<Props> = ({
     };
 
     return (
-        <AutosuggestDeprecated
-            getSuggestions={suggestionsList}
+        <Autosuggest
+            suggestions={suggestionsList}
+            onSelectionAdd={handleSelection}
+            selectedSuggestions={suggestionsList}
             suggestionToString={suggestionToString}
             isLoading={isLoading}
             isFocused={isFocused}
-            defaultInputValue={defaultInputValue}
+            defaultValue={defaultInputValue}
             inputPlaceholder={inputPlaceholder}
             showClearButton={singleLocation}
             clearTitle={clearTooltipLabel}
             noSuggestionsPlaceholder={noSuggestionsPlaceholder}
-            listRenderer={renderListPoweredByGoogle}
-            saveSelectedValueToInput={singleLocation}
```

Comment:
Did you test LocationSelector? Does it return the same output?

Reply:
Yes

Reply:
I meant to add the answer to my question :)